AlwaysOnKotlin/compose/src/main/java/com/example/android/wearable/wear/alwayson/AlwaysOnApp.kt

```diff
@@ -173,6 +175,10 @@ fun AlwaysOnApp(
                     val triggerTime = currentInstant.getNextInstantWithInterval(
                         AMBIENT_INTERVAL
                     )
+                    // setExact() should *not* be used in production code. In a real application,
```

Comment:
@garanj @yschimke Is this accurate?

Reply:
Lets not show it then.  Let's remove it.

Either say
a) Leave it as the default behaviour on Wear 3 + 4
b) Explain the difference and Wear 4 doesn't allow.

We shouldn't show working around platform efficiency restrictions.